js/src/forum/components/Search.js

```diff
@@ -211,16 +189,10 @@ export default class Search extends Component {
   }
 
   /**
-   * Clear the search input and the current controller's active search.
+   * Clear the search
    */
   clear() {
-    this.value('');
-
-    if (this.getCurrentSearch()) {
-      app.current.clearSearch();
-    } else {
-      m.redraw();
-    }
+    this.clear();
```

Comment:
That's not going to go well. :stuck_out_tongue_winking_eye:

Reply:
:facepalm: